lib/crossdoc/tree.rb

```diff
@@ -36,11 +36,25 @@ def download
         elsif @src.index('./')==0
           @io = open(@src.gsub('./', Dir.pwd + '/'))
         else # assume it's a URL
-          @io = URI.open(@src)
+          @io = URI.open @src
+          if @src.include?('logo_images') || num_images < 6
```

Comment:
I think hard coding 'logo_images' is overreaching crossdoc's purview a bit here. I would say it is fine to limit the width/quality of logo images as long as we're not too zealous about the quality setting. Maybe it'll become necessary to have a special case for logo images but ideally it wouldn't be.

Reply:
I thought it was necessary to get around an issue I was seeing in development, but I've removed it and it still works, so I guess not.